travis/before-cache-gradle

```diff
@@ -5,3 +5,9 @@ caches="$HOME/.gradle/caches"
 rm -rf \
    "$caches/modules-2/modules-2.lock" \
```

Comment:
Remove, since this is now redundant with respect to the `*.lock` deletion performed by one of the `find` commands below.

Reply:
Fixed